fl4health/losses/fenda_loss_config.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple
+
+import torch
+
+from fl4health.losses.contrastive_loss import ContrastiveLoss
+from fl4health.losses.cosine_similarity_loss import CosineSimilarityLoss
+from fl4health.losses.perfcl_loss import PerFclLoss
+
+
+class PerFclLossConfig:
+    def __init__(
+        self,
+        device: torch.device,
+        global_feature_contrastive_loss_weight: float,
+        local_feature_contrastive_loss_weight: float,
+        global_feature_loss_temperature: float = 0.5,
+        local_feature_loss_temperature: float = 0.5,
+    ) -> None:
+        self.global_feature_contrastive_loss_weight = global_feature_contrastive_loss_weight
+        self.local_feature_contrastive_loss_weight = local_feature_contrastive_loss_weight
+        self.perfcl_loss_function = PerFclLoss(device, global_feature_loss_temperature, local_feature_loss_temperature)
+
+
+class CosSimLossConfig:
```

Comment:
nit: Mine as well expand this out to match format of corresponding loss like we have for the others. CosineSimilarityLossConfig